src/stats.h

```diff
@@ -12,64 +12,109 @@ constexpr uint32_t kPopulationSize = 5450000;
 constexpr uint32_t kDecadesCount = 9;
 constexpr std::array<double, kDecadesCount> kPopulationAge = {0.11, 0.10, 0.12, 0.16, 0.15,
                                                               0.13, 0.13, 0.07, 0.03};
-const double kLog2 = std::log(2);
+constexpr std::array<double, kDecadesCount> kDeathProbabilities = {
+    0.002, 0.002, 0.002, 0.002, 0.004, 0.013, 0.036, 0.08, 0.148};
+constexpr double kDeathThreshold = 0.5;
+constexpr uint32_t kSymptomsLength = 28;
+constexpr uint32_t kFactorialLength = 5000;
 
-auto beta_distribution(double alpha, double beta, std::mt19937* gen) -> double {
-  std::gamma_distribution<> X(alpha, 1), Y(beta, 1);
-  double x = X(*gen), y = Y(*gen);
-  return x / (x + y);
-}
+// Class containing all the logic for generating people. Each person:
+// * is generated according to population age distribution
+// * has symptoms' severity generated based on their age
+// * has disease length generated based on symptoms severity
+class Stats {
```

Comment:
Minor: Maybe it should be called `PopulationDistribution` or `PopulationModel` or something similar.

Reply:
Right, `PopulationModel` sounds good. The name of the files `stats.h` was more fitting earlier.